spec/std/process_spec.cr

```diff
@@ -268,7 +268,7 @@ describe Process do
     end
 
     describe "does not execute batch files" do
-      %w[.bat .Bat .BAT .cmd .cmD .CmD].each do |ext|
+      {".bat", ".Bat", ".BAT", ".cmd", ".cmD", ".CmD", ".bat ", ".cmd ...", ".bat. ."}.each do |ext|
         it ext do
           with_tempfile "process_run#{ext}" do |path|
```

Comment:
Should this change since the extensions can now include spaces and periods?

Reply:
Sounds good. However, we should keep it an array, not a tuple.
Alternatively, we could keep the string array literal and escape the spaces that are not meant as delimiters.
```suggestion
      %w[.bat .Bat .BAT .cmd .cmD .CmD .bat\ .cmd\ ... .bat.\ .].each do |ext|
```

Reply:
Will do! My review comment is specifically about this line:
`with_tempfile "process_run#{ext}" do |path|`

Wouldn't it result into `process_run.bat. .`?

Reply:
I'll leave it for someone else as I don't want to break anything in the process (plus I feel like this might catch any unexpected errors like `File::NotFoundError`)